lib/src/localizations.dart

```diff
@@ -156,10 +156,10 @@ class InsporationLocalizations {
     locale: localeName
   );
 
-  String get errorSignInNoHost => Intl.message(
-      'Unable to resolve host, are you sure this pod supports the API?',
-      name: 'errorSignInNoHost',
-      desc: 'Error message if no host can not be resolved',
+  String get errorInvalidAPI => Intl.message(
```

Comment:
```suggestion
  String errorAuthorizationFailed(String user) => Intl.message(
```

Adding the argument also requires `args` and `examples`.